swagger.yaml

```
@@ -215,6 +215,38 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/forecastWebhook:
+    post:
+      operationId: forecastWebhook
+      summary: Handle deleted time entries from Forecast API
+      description: This endpoint handles webhook notifications from Forecast.it API
+      tags:
+        - ForecastWebhook
+      parameters:
+        - in: query
+          name: forecastWebhookKey
+          required: false
```

```suggestion
          required: true
```
As discussed within the API implementation PR.